.circleci/config.yml

```diff
@@ -315,15 +315,15 @@ workflows:
             - slack
       - test-chart-ams:
           requires:
-            - deploying-bpmns
+            - test-chart-gov
           context:
             - AWS
             - Helm
             - slack
             - Ngrok
       - test-chart-gov:
           requires:
-            - test-chart-ams
```

Comment:
Please don't change this order

Reply:
done it temporarily to test, will reverse and notify.

Reply:
reversed